Added Cyro

Havaticis opened the pull request:
Added Cyro to your script, hopefully I did it right.

Havaticis commented:
First one was the incorrect file. (has the Dragoon q and e in it) The second one is just the Addition of the Cyro.

TheBestPessimist commented on StarCraft2.ahk:
```diff
@@ -130,7 +137,7 @@ class SC2
         this.tankyPos := yPos
         this.casterxPos := xPos + 188    ; compared to the center: a little bit to the right
-        this.casteryPos := yPos - 120    ; compared to the center: a little bit upper
+        this.casteryPos := yPos - 25    ; compared to the center: a little bit upper
```
That doesn't need to change. You want to fire his "stun rocket thingy" behind you a little, so that zombies will come closer to you.
You want that (zombies coming closer to you) because the closer they are to you, the more of them will get hit by goon q, or my medic grenade (think that mouse pos {X, Y} represents the center of the aoe -- you want as many zombies hit by that aoe spell as possible).

TheBestPessimist replied:
I think it's better to add a new pair of coordinates specifically for these defensive spells. (something i will do after merging this PR)

TheBestPessimist commented on StarCraft2.ahk:
```diff
@@ -25,6 +25,8 @@
 
 ; CapsLock & Numpad0: toggle Templar
 
+; CapsLock & Numpad6: toggle Cyro
```
This comment should go a few rows lower: just below `; CapsLock & Numpad7: toggle Medic`, because i have ordered them by `Numpadx` decreasingly.

Havaticis commented:
I've revised my changes. I've reverted your original caster position y axis and added a specific one for Cyro only, and it's now perfect.  Also I have increased the time between Medic commands as I didn't see the need for one ever 2 seconds and it drains non-majors of energy too quickly.

Havaticis commented:
Going to try and fix it
